tests/execution/call-mutation.md

```diff
@@ -0,0 +1,85 @@
+# Call mutation
+
+#### server:
+
+```graphql
+schema @server @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
+  query: Query
+  mutation: Mutation
+}
+
+input PostInput {
+  body: String
+  title: String
+  userId: Int
+}
+
+type Mutation {
+  insertPost(input: PostInput): Post @http(body: "{{args.input}}", method: "POST", path: "/posts")
+}
+
+type Post {
+  body: String
+  id: Int
+  title: String
+  userId: Int
+}
+
+type Query {
+  firstUser: User @http(method: "GET", path: "/users/1")
+}
+
+type User {
+  id: Int
+  name: String
+  post: Post
+    @call(mutation: "insertPost", args: {input: "{\"body\":\"user-body\",\"title\":\"user-title\",\"userId\":1}"})
```

Comment:
Writing the input as string encoded JSON isn't the most convinient. I think we should be able to pass the value using the graphQL syntax as follows:
```graphql
@call(mutation: "insertPost", args: {input: {body: "user-body", title: "user-title", userId: 1}})
```